packages/algob/sample-project/algob.config.js

```diff
@@ -8,29 +8,43 @@
 //   addr: "KFMPC5QWM3SC54X7UWUW6OSDOIT3H3YA5UOCUAE2ABERXYSKZS5Q3X5IZY",
 //   mnemonic: "call boy rubber fashion arch day capable one sweet skate outside purse six early learn tuition eagle love breeze pizza loud today popular able divide"
 // }]);
-
+const { loadKMDAddresses } = require("algob");
+module.exports = (async function() {
 // Example: accounts constructed using the Account object
 let accounts = [{
   name: "owner",
   addr: 'UDF7DS5QXECBUEDF3GZVHHLXDRJOVTGR7EORYGDBPJ2FNB5D5T636QMWZY',
   sk: new Uint8Array([28,  45,  45,  15,  70, 188,  57, 228,  18,  21,  42, 228,  33, 187, 222, 162,  89,  15,  22,  52, 143, 171, 182,  17, 168, 238,  96, 177,  12, 163, 243, 231, 160, 203, 241, 203, 176, 185,   4,  26,  16, 101, 217, 179, 83, 157, 119,  28,  82, 234, 204, 209, 249,  29,  28, 24,  97, 122, 116,  86, 135, 163, 236, 253])
 }]
 
+// KMD credentials
+let kmdconfig = {
+  host: "127.0.0.1",
+  port: 7833,
+  token: "09c2da31d3e3e96ed98ba22cc4d58a14184f1808f2b4f21e66c9d38f70ca7232"
+}
+
+let wallet = {
+  name: "MyTestWallet1",
+  password: "testpassword"
+}
 // const { loadAccountsFromFileSync } = require("algob");
 // // Example: accounts loaded from a file:
 // const accFromFile = loadAccountsFromFileSync("assets/accounts_generated.yaml");
 // accounts = accounts.concat(accFromFile);
-
+//console.log("OK");
 let defaultCfg = {
   host: "http://localhost",
   port: 8080,
   token: "content_of/algorand-node-data/algod.token",
   accounts: accounts,
+  kmdAddresses: await loadKMDAddresses(kmdconfig.host, kmdconfig.token, kmdconfig.port, 
```

Comment:
do we need to use `await` here?

If `kmdAddresses` will be a promise then it's ok, and we will not have any `async / await` in the config and everything will be simpler.

Reply:
Also, let's rename this to `kmdAccounts`

Reply:
if you are using `await` then the value here is not promise (await resolve promises). If we don't use await then it should be fine, and we will do await in other parts of the code.

Reply:
yes i did this without await first, but that didn't work, i was getting a blank array,
I added await config.kmdaddresess in node-info
and then ran node-info from cli

Reply:
when we load the algob.config.js without resolving promise it stores a blank array in kmdAddresses, 
and even after adding await in node-info , it results in blank array